draft-ietf-quic-transport.md

```diff
@@ -889,23 +908,24 @@ Packets protected with 0-RTT keys use a type value of 0x7C.  The connection ID
 fields for a 0-RTT packet MUST match the values used in the Initial packet
 ({{packet-initial}}).
 
-The client can send 0-RTT packets after receiving an Initial
-{{packet-initial}} or Handshake ({{packet-handshake}}) packet, if that
+The client can send further 0-RTT packets after receiving an Initial
+({{packet-initial}}) or Handshake ({{packet-handshake}}) packet, if that
 packet does not complete the handshake.  Even if the client receives a
 different connection ID in the Handshake packet, it MUST continue to
 use the same Destination Connection ID for 0-RTT packets, see
 {{connection-id-encoding}}.
 
-The version field for protected packets is the current QUIC version.
+The version field for protected packets contains the current QUIC version.
 
 The packet number field contains a packet number, which has additional
 confidentiality protection that is applied after packet protection is applied
 (see {{QUIC-TLS}} for details).  The underlying packet number increases with
 each packet sent, see {{packet-numbers}} for details.
 
-The payload is protected using authenticated encryption.  {{QUIC-TLS}} describes
-packet protection in detail.  After decryption, the plaintext consists of a
-sequence of frames, as described in {{frames}}.
+The content of 0-RTT and 1-RTT packets is protected using authenticated
```

Comment:
I believe everything but INITIAL and RETRY are authenticated, so I think this change is not an improvement.